charter.md

```diff
@@ -35,3 +35,18 @@ The Docs Working Group will focus on the efforts described below, in service of
         - Automated discourse or CI bots for gathering user feedback from community sites
         - Adding a new embedded "Hot Topics" feature on ReadTheDocs FAQ pages to automated discourse community discussion aggregator
         - Any other innovations or work that may improve Jovyans' experience in/understanding of the Jupyter ecosystem
+
+Founding Members
+
+- Carlos Brandt
+- Eric Gentry
+- Frederic Collonval
+- Mike Krassowski
+- Nick Bollweg
+- Rosio Reyes
```

Comment:
Please add  me as a founding member. Thanks.

Reply:
@ivanov and @Carreau if they have interest as well.

Reply:
Thanks, 

While I have interest and I'm still actively working on [papyri](https://github.com/jupyter/papyri) which is still part of a CZI grant for the next few month, I don't have much time and don't want to guaranty I can participate in all that.

Papyri definitely fit into all the "cross-cutting" area and I'd love to get some help from this committee to steer it, or get it integrated into JupyterLab and the other Jupyter projects.

Reply:
> Please add me as a founding member. Thanks.

Absolutely! I will add @ivanov (and @Carreau I can still add you if you'd like, don't feel obligated to spend a ton of time as partial participation is still valuable, let me know).

> Which also begs the question: are we missing others on this list as well? I would definitely like to see Chris Holgraf to be part of this group and Tania Allard as well if they have interest.

This would be GREAT, please reach out to anyone you can think of who might be interested, we would love to have their thoughts and experience to help us do our work :) I will reach out to @choldgraf on discourse.

Reply:
Just added @willingc and @ivanov, you should see the updates now!